conf/base/globals.yml

```
@@ -1,7 +1,8 @@
 n_qubits: 4
 n_layers: 22
 vqc_ansatz: "circuit_19"
-iec_ansatz: "spread_layers"
+iec_ansatz: "default" #spread_qubits
```

If I remember correctly, you documented this already in another PR. So please consider removing the comment in the back as it is not needed.

```suggestion
iec_ansatz: "default"
```